### PR TITLE
Loki Monaco Editor: add component tests

### PR DESCRIPTION
**Pull request by matyax**
**What this PR does / why we need it**:

This PR adds missing tests for the components added in the initial PR.

According to https://github.com/suren-atoyan/monaco-react/issues/88, there's no trivial way to render the actual Monaco editor in the test, or to mock it. So adding some simple test to have at least some minimum converage.

**Which issue(s) this PR fixes**:

Part of https://github.com/grafana/grafana/issues/44985

**Special notes for your reviewer**:

### CODE REVIEW

File: public/app/plugins/datasource/loki/components/monaco-query-field/MonacoFieldWrapper.test.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import { render, screen } from '@testing-library/react';
```
Comment (matyax) [inline public/app/plugins/datasource/loki/components/monaco-query-field/MonacoFieldWrapper.test.tsx:1]: @gabor I see we have https://github.com/grafana/grafana/blob/main/packages/grafana-ui/src/components/Monaco/ReactMonacoEditorLazy.tsx , so maybe we don't need to do lazy loading here and in Prometheus?

**Review by ivanahuckova — APPROVED** [review]
LGTM. By the way, because the next minor release is 9.3 and you are not backporting, the milestone should be 9.3.0

File: public/app/plugins/datasource/loki/components/monaco-query-field/MonacoFieldWrapper.test.tsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+import { createLokiDatasource } from '../../mocks';
+
+import { MonacoQueryFieldWrapper, Props } from './MonacoQueryFieldWrapper';
+
+function renderComponent({
+  initialValue = '',
+  onChange = jest.fn(),
+  onRunQuery = jest.fn(),
+  runQueryOnBlur = false,
+}: Partial<Props> = {}) {
```
Comment (ivanahuckova) [inline public/app/plugins/datasource/loki/components/monaco-query-field/MonacoFieldWrapper.test.tsx:14]: 1 thing I like to do is not export component `Props`, if we **only** need them in test file and use ComponentProps instead  `Partial<ComponentProps<typeof MonacoQueryFieldWrapper>>`. 

Here is an example:
https://github.com/grafana/grafana/blob/main/public/app/features/inspector/InspectDataTab.test.tsx#L13

Comment (matyax) [reply public/app/plugins/datasource/loki/components/monaco-query-field/MonacoFieldWrapper.test.tsx:14]: True, good suggestion.

File: public/app/plugins/datasource/loki/components/monaco-query-field/MonacoFieldWrapper.test.tsx (hunk 1 above)
Comment (gabor) [reply public/app/plugins/datasource/loki/components/monaco-query-field/MonacoFieldWrapper.test.tsx:1]: @matyax probably yes.
i doublechecked with the browser-dev-tools the current flow:
1. i created a dashboard, added a loki datasource with a query, saved it
2. went back to the dashboard-webpage
3. opened the browser-devtools to see all files loaded
4. did a full reload with cache disabled
5. in the files-that-were-downloaded, i can see the `lokiPlugin` (`2.4MB`). that is ok, we need the loki datasource to display the dashboard. but no monaco, because we do not need the loki-query-field
6. i went into edit-mode for the dashboard-panel.
7. now we loaded `loki-query-field` (`1.2MB`), and monaco's `editor.main.js` (`3.1MB`)

if we remove the lazy-thing, i assume the `loki-query-field` will get bundled with `lokiPlugin`, but maybe they have some shared code so the two of them together might occupy less space.

in other words, i agree, we should remove that lazy-load, but please check the file-sizes afterwards, to be sure everything is OK. the most important thing is that the `3MB` monaco-javascript is not loaded together with `lokiPlugin`. 👍
